AsyncDisplayKit/ASImageNode.mm

```diff
@@ -305,7 +305,7 @@ - (void)displayDidFinish
     
     ASDN::MutexLocker l(_imageLock);
     _displayCompletionBlock = nil;
-    ASDN::MutexLocker u(_imageLock);
```

Comment:
In both of these cases, I would prefer that we use the more common pattern that I recall seeing before -- .lock() and .unlock() calls directly on the _imageLock.  This will also reduce the overhead of these stack allocated objects, which although low, is consistently measurable in hot code like this.